draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1386,9 +1386,19 @@ Content-Type: text/plain
 <t>
    In general, a client &MUST; verify the service identity using the
    verification process defined in
-   <xref target="RFC6125" x:fmt="of" x:sec="6"/> (for a reference identifier of
-   type URI-ID) unless the client has been specifically configured to accept
-   some other form of verification. For example, a client might be connecting
+   <xref target="RFC6125" x:fmt="of" x:sec="6"/>. The client &MUST; construct
+   a reference identity that is a DNS-ID if the URI contains a hostname and an
+   IP-ID if the hostname was a literal IP address. Validing an IP-ID is
+   defined in <xref target="https.ip-id"/>.
```

Comment:
```suggestion
   <xref target="RFC6125" x:fmt="of" x:sec="6"/>. The client &MUST; construct
   a reference identity from the service's host: if the host is a literal IP address
   (<xref target="https.ip-id"/>), the reference identity is an IP-ID, otherwise
   the host is a name and the reference identity is a DNS-ID.
```